decidim-core/lib/decidim/seeds.rb

```diff
@@ -9,6 +9,10 @@ module Decidim
   class Seeds
     protected
 
+    def ci_environment?
+      Decidim::Env.new("CI").present? && Rails.env.test?
+    end
```

Comment:
I think I have a better alternative for this one: on some environments (for instance, Gitpod) we actually want to improve the generation time of the seeds and not have that many data. 

```suggestion
    def fast_seeds?
      Decidim::Env.new("FAST_SEEDS").present?
    end
```

Reply:
fixed by [e977e58](https://github.com/decidim/decidim/pull/14528/commits/e977e589cc7604b887cb0cf8a0f4cabef03d79dd)